packages/backend/src/apps/google-docs/actions/create-new-doc/index.js

```diff
@@ -0,0 +1,60 @@
+import defineAction from '../../../../helpers/define-action.js';
+
+export default defineAction({
+  name: 'Create new document',
```

Comment:
```suggestion
  name: 'Create document',
```

Let's also update the folder name with `create-document` to reflect the key in its convention.